themes/custom/droopler_theme/scss/components/_d_p_side_by_side.scss

```diff
@@ -76,13 +78,64 @@ $d-p-side-by-side-node-box-margin-x: 0 - $grid-gutter-width / 2 !default;
 
     .d-p-text-block {
       margin-bottom: 0;
+      h2 {
+        font-size: $font-main-header-size;
+      }
     }
+
+    .field--name-field-d-cta-link {
+      a {
+        @extend .btn;
+        @extend .btn-danger;
+        @extend .mt-4;
+      }
+    }
+
   }
 
   .item-1 {
     .user-background-defined {
       &::before {
-        left: unset;
+        //left: unset;
+        //right: 0;
+      }
+    }
+  }
+  .item-1,
+  .item-2 {
+    .user-image-background {
+      padding: $d-p-side-by-side-image-background-padding-y 0;
+      &:before {
+        content: none;
+      }
+      .image-background-container {
+        position: absolute;
+        height: 100%;
+        top: 0;
+        right: 0;
+        display: block;
+        z-index: -1;
+        &.background-properties {
+          -webkit-background-size: cover;
+          -moz-background-size: cover;
+          -o-background-size: cover;
```

Comment:
the prefixes like `-webkit-` are not necessary, they will be added automagically during compile